connection_notifier.cc

```diff
@@ -55,9 +55,3 @@ future<> notify_disconnected_client(gms::inet_address addr, client_type ct, int
                      db::system_keyspace::clients_cf_name());
     return db::execute_cql(req, addr.addr(), port, to_string(ct)).discard_result();
 }
-
```

Comment:
This shouldn't be here at all. You're adding the function in previous patch and then removing it here. Just don't add the method at all please.